extensions/adobe/experience/marketo/marketo-lead.schema.json

```diff
@@ -0,0 +1,79 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/marketo/marketo-lead",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Marketo Lead Schema",
+  "type": "object",
+  "description": "Marketo Lead schema. Includes other Profile mixins like person/work details, opt in/out etc.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/context/profile-work-details",
+    "https://ns.adobe.com/xdm/context/profile-personal-details",
+    "https://ns.adobe.com/xdm/context/profile-person-details",
+    "https://ns.adobe.com/xdm/context/profile-owning-entities",
+    "https://ns.adobe.com/xdm/context/profile-lead-crm",
+    "https://ns.adobe.com/xdm/context/optinout"
+  ],
+  "definitions": {
+    "marketo-lead": {
+      "properties": {
+        "marketo:leadID": {
+          "title": "Marketo Lead ID",
+          "description": "Marketo lead id.",
+          "type": "string"
+        },
+        "marketo:marketoPersonID": {
```

Comment:
marketo:ContactID, marketo:AccountID are also needed to establish a chain of relationship.